src/extensions/dicom-file-format.md

```diff
@@ -0,0 +1,60 @@
+Importantly, to prevent isolated development within BIDS, it is crucial to introduce support for a data format that is widely used in a vast domain of clinical research and is actively being developed.
+
+### Current State & Prior Work
+
+Ongoing projects related to DICOM and BIDS ecosystems include:
+
+- Numerous working groups (WGs), similar to our BEPs, supporting new data modalities: https://www.dicomstandard.org/activity/wgs .
+We have contacted WG-16 (#1515)  and identified potential work items (#1516 and so on).
+- DICOM is transitioning from "free text" to existing dictionaries and ontologies for various fields, e.g. SNOMED CT for Body Part ([table_CID_8134](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/sect_CID_8134.html#table_CID_8134)
+and [chapter_L](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html)).
+- Data archives such as the [NCI Imaging Data Commons](https://imaging.datacommons.cancer.gov/) operate on DICOMs
+  - They have developed libraries like [highdicom](https://github.com/ImagingDataCommons/highdicom) to "facilitate the creation and handling of DICOM objects for image-derived information".
+  - There likely more developments to benefit from reusing or integrating with, e.g. https://mhub.ai
+- We are already using many DICOM tags and aim to harmonize this usage further (#1450). Given DICOM's aforementioned shift to using ontologies, we are already lagging here, and in turn, we might want to contribute back perhaps regarding the NIDM-Terms effort (attn @dbkeator).
+- Projects such as the http://chrisproject.org on the border of clinical and research communities provide infrastructure centered around data digestion from PACS.
+- ... Please contribute as suggestions ...
+
+As part of the https://github.com/matthew-brett/czi-nibabel grant work, @matthew-brett led a series of journal club meetings in late 2021 to review different possible file formats for imaging data.
+
+@matthew-brett - do you have recordings/notes from those sessions? I don't recall DICOM being considered.
+
+### Summary of Proposed Developments
+
+Develop a BEP to allow `.dcm` to be an accepted file format for MRI and other data types (the BEP work would decide which specifically).
+
+#### Cons: Potential Issues and Solutions
+
+- Duplication of metadata between `.dcm` and BIDS-sidecar `.json`.
+  - This issue is not new or specific to DICOM: it needs to be addressed conceptually in BIDS (https://github.com/bids-standard/bids-specification/pull/761).
+  - In fact, it offers a **benefit** of metadata harmonization, particularly when DICOM or manufacturers do not yet provide information in non-manufacturer-specific sections of DICOM.
+- Libraries needed to support DICOM I/O to support BIDS: experiences with `dcm2niix` suggest that it remains a challenge to consistently access imaging data from DICOMs.
```

Comment:
@neurolabusc please have a look at this BEP idea and please express yourself -- your experience and knowledge would be of the most value here!

Reply:
I am agnostic about this. The issue with DICOM is that it is like the English language, which is used differently by different individuals (e.g. `pants` means something different in the USA than the UK). Each manufacturer has its own definitions, can use terms like `RepetitionTime` differently, and uses private tags for a lot of details. This seems like an aspirational goal, but it seems like the initial work must come from harmonization efforts by the DICOM work groups.

Assuming a user wants to make a DICOM-based BIDS dataset, how does the BIDS validator work? Consider a Philips fMRI time series, where that manufacturer has no method for defining slice timing. What tool will be used to inject required details not in the raw DICOM? Do we need to define a generic set of tags to define these (presumably with the DICOM work group assigning a public tag)?

I think that dcm2niix (and dicm2nii, SPM) face the red queen problem: we must always update our code as the manufacturers change their DICOM implementation and our understanding of their usage evolves. Adopting DICOM simply shifts the burden of perpetual updating to other tools.

However, I also respect the [view](https://peerj.com/articles/2057/) of @fedorov and his [dcmqi](https://github.com/qiicr/dcmqi) does deliver on the promise of using DICOM directly for science. He started with the DICOM format, while many of the BIDS pipelines are wrappers for tools that do not support DICOM. Therefore, this would require a lot of investment.